src/results/chp.jl

```diff
@@ -54,64 +60,70 @@ function add_chp_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
 	else
 		CHPtoBatt = zeros(length(p.time_steps))
 	end
-	r["year_one_to_battery_series_kw"] = round.(value.(CHPtoBatt), digits=3)
+	r["electric_to_battery_series_kw"] = round.(value.(CHPtoBatt), digits=3)
 	@expression(m, CHPtoLoad[ts in p.time_steps],
 		sum(m[Symbol("dvRatedProduction"*_n)][t, ts] * p.production_factor[t, ts] * p.levelization_factor[t]
 			for t in p.techs.chp) - CHPtoBatt[ts] - CHPtoGrid[ts])
-	r["year_one_to_load_series_kw"] = round.(value.(CHPtoLoad), digits=3)
+	r["electric_to_load_series_kw"] = round.(value.(CHPtoLoad), digits=3)
 	# Thermal dispatch breakdown
     if !isempty(p.s.storage.types.hot)
 		@expression(m, CHPtoHotTES[ts in p.time_steps],
 			sum(m[Symbol("dvProductionToStorage"*_n)]["HotThermalStorage",t,ts] for t in p.techs.chp))
 	else 
 		CHPtoHotTES = zeros(length(p.time_steps))
 	end
-	r["year_one_thermal_to_tes_series_mmbtu_per_hour"] = round.(value.(CHPtoHotTES / KWH_PER_MMBTU), digits=5)
+	r["thermal_to_tes_series_mmbtu_per_hour"] = round.(value.(CHPtoHotTES / KWH_PER_MMBTU), digits=5)
 	@expression(m, CHPThermalToWasteKW[ts in p.time_steps],
 		sum(m[Symbol("dvProductionToWaste"*_n)][t,ts] for t in p.techs.chp))
-	r["year_one_thermal_to_waste_series_mmbtu_per_hour"] = round.(value.(CHPThermalToWasteKW) / KWH_PER_MMBTU, digits=5)
+	r["thermal_curtailed_series_mmbtu_per_hour"] = round.(value.(CHPThermalToWasteKW) / KWH_PER_MMBTU, digits=5)
     if !isempty(p.techs.steam_turbine) && p.s.chp.can_supply_steam_turbine
         @expression(m, CHPToSteamTurbineKW[ts in p.time_steps], sum(m[Symbol("dvThermalToSteamTurbine"*_n)][t,ts] for t in p.techs.chp))
     else
         CHPToSteamTurbineKW = zeros(length(p.time_steps))
     end	
-    r["year_one_thermal_to_steamturbine_series_mmbtu_per_hour"] = round.(value.(CHPToSteamTurbineKW) / KWH_PER_MMBTU, digits=5)
+    r["thermal_to_steamturbine_series_mmbtu_per_hour"] = round.(value.(CHPToSteamTurbineKW) / KWH_PER_MMBTU, digits=5)
     @expression(m, CHPThermalToLoadKW[ts in p.time_steps],
         sum(m[Symbol("dvThermalProduction"*_n)][t,ts] + m[Symbol("dvSupplementaryThermalProduction"*_n)][t,ts]
             for t in p.techs.chp) - CHPtoHotTES[ts] - CHPToSteamTurbineKW[ts] - CHPThermalToWasteKW[ts])
-    r["year_one_thermal_to_load_series_mmbtu_per_hour"] = round.(value.(CHPThermalToLoadKW) / KWH_PER_MMBTU, digits=5)
-    r["year_one_chp_fuel_cost_before_tax"] = round(value(m[:TotalCHPFuelCosts] / p.pwf_fuel["CHP"]), digits=3)                
+    r["thermal_to_load_series_mmbtu_per_hour"] = round.(value.(CHPThermalToLoadKW) / KWH_PER_MMBTU, digits=5)
+    
+	@expression(m, CHPThermalProdKW[ts in p.time_steps],
+		sum(m[Symbol("dvThermalProduction"*_n)][t,ts] + m[Symbol("dvSupplementaryThermalProduction"*_n)][t,ts] - 
+        m[Symbol("dvProductionToWaste"*_n)][t,ts] for t in p.techs.chp))
+	r["thermal_production_series_mmbtu_per_hour"] = round.(value.(CHPThermalProdKW) / KWH_PER_MMBTU, digits=5)
```

Comment:
@Bill-Becker could you please double-check this added calc for `thermal_production_series_mmbtu_per_hour`?

Reply:
Yep it looks right, but it also looks like a duplicate expression from the above "Year1CHPThermalProdKWH" expression. Right?

Reply:
Well, the Year1CHPThermalProdKWH expression was a scalar and the CHPThermalProdKW a time series, but yes definitely close enough to be duplicative. I consolidated in https://github.com/NREL/REopt.jl/pull/160/commits/91e37859fbe0b27b2d1080ea294f49811f27f8e5 and https://github.com/NREL/REopt.jl/pull/160/commits/d02e17055d1cafd494204c3bafc2be35d558737f

Reply:
Good point, I missed that, but thanks for consolidating!